subworkflows/primaryclipanalysis.nf

```diff
@@ -100,39 +151,88 @@ workflow PRIMARY_CLIP_ANALYSIS {
     STAR_SAMTOOLS_INDEX ( STAR_ALIGN.out.bam_sorted )
     ch_umi_input = STAR_ALIGN.out.bam_sorted.combine(STAR_SAMTOOLS_INDEX.out.bai, by: 0)
     UMITOOLS_DEDUP ( ch_umi_input )
-
     UMITOOLS_SAMTOOLS_INDEX ( UMITOOLS_DEDUP.out.bam )
+    reads.map{triplet -> [
+        triplet[0], file(triplet[2] + "/SAMTOOLS_FAIDX/*.fa.fai")
+    ]}.set{ ch_genome_fa }
```

Comment:
Tiny detail, but should be `ch_genome_fai`

Reply:
Fixed!